manifest_staging/charts/workload-identity-webhook/templates/azure-wi-webhook-controller-manager-deployment.yaml

```diff
@@ -74,6 +74,7 @@ spec:
         resources:
           {{- toYaml .Values.resources | nindent 10 }}
         securityContext:
+          {{- toYaml .Values.securityContext | nindent 10 }}
```

Comment:
The values already exist in the yaml and doing this will not work. Based on my testing for duplicate values in the yaml, the latter one takes preference.